src/sql/azdata.d.ts

```diff
@@ -1879,7 +1879,10 @@ declare module 'azdata' {
 	export interface BackupConfigInfo {
 		recoveryModel: string;
 		defaultBackupFolder: string;
-		backupEncryptors: {};
+		backupEncryptors: {
```

Comment:
Based on how we were using this in our code, the typing wasn't right. So I changed it to be the type we are using in the code.